docs/src/components/Sidebar/Sidebar.tsx

```diff
@@ -465,6 +465,7 @@ class Sidebar extends React.Component<any, any> {
           </a>
           <Input
             styles={topItemTheme}
+            variables={siteVars => ({ backgroundColor: siteVars.colorScheme.default.background3 })}
```

Comment:
```suggestion
            inverted
```

I was able to get the same result with `inverted` prop, #1684.

![image](https://user-images.githubusercontent.com/14183168/63443331-e290e180-c434-11e9-8509-a5dc705ba8e9.png)